File: android studio/app/src/main/java/android/fit/ba/hotelwien/data/GetSobeResultVM.java
package android.fit.ba.hotelwien.data;

import java.io.Serializable;
import java.util.List;

public class GetSobeResultVM implements Serializable {

    public static class Row implements Serializable {
        public int SobaID;
        public String NazivSobe;
        public String Cijena;
        public  String DatumPocetka;
        public String DatumZavrsetka;
    }
    public String ImaPodataka;
    public List<Row> rows;

}

File: android studio/Web Servisi/HotelWIEN_API/ModelVM/GetSobeResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace HotelWIEN_API.ModelVM
{
    public class GetSobeResultVM
    {
        public List<Row> rows { get; set; }
        public string ImaPodataka { get; set; }
        public class Row
        {
            public int SobaID { get; internal set; }
            public string NazivSobe { get; internal set; }
            public decimal Cijena { get; internal set; }
            public string DatumPocetka { get; internal set; }
         
            public string DatumZavrsetka { get; internal set; }
        }
    }
}